compute/instances/how-to/create-an-instance.mdx

```diff
@@ -31,29 +31,28 @@ Select a tab below for instructions on how to create an Instance via either our
           <Message type="important">
-             - Ensure that a volume with an OS image has a minimum capacity of 10 GB. For a GPU OS, the recommended size is 125 GB.
+             - Ensure that the volume containing your OS image has a minimum size of 10 GB. For a GPU OS, the recommended size is 125 GB.
              - When multiple Block Storage volumes are linked to your Instance, the primary volume will host the OS and is essential for booting the Instance. Once the Instance is created can [modify your boot volume](/compute/instances/how-to/use-boot-modes/#how-to-change-the-boot-volume).
              - Booting from a volume that either lacks an OS or is among multiple volumes with identical operating systems can lead to inconsistent boot outcomes.
           </Message>
-        - Configure the network of the Instance. You can either select to use **Routed public IP** (a dedicated public IP address routed to your Instance that allows direct communication between the Instance and the Internet) or a **NAT public IP** (a public IP address that uses a carrier-grade NAT to translate the Instances NAT IP address). If you are unsure which to use, we recommend a routed public IP for ease of use and improved performance.
+        - **Configure network** of the Instance.
           - Leave the checkbox ticked to assign a **Public IPv4** to the Instance. You can either allocate a new IPv4 address or select one or multiple existing IPv4s. Alternatively, uncheck the box if you do not want an IPv4.
           - Leave the checkbox ticked to assign a **Public IPv6** to the Instance. You can either allocate a new IPv6 address or select one or multiple existing IPv6s. Alternatively, uncheck the box if you do not want an IPv6.
             <Message type="tip">
-              You can attach up to 5 IPs to an Instance, combining IPv4 and IPv6 addresses.
+              You can attach up to 5 IPs to an Instance, combining IPv4 and IPv6 addresses, which is useful for running different services or applications on the same Instance.
             </Message>
-        - Enter a **Name** for your Instance, or leave the randomly-generated name in place. Optionally, you can add [tags](/compute/instances/concepts/#tags) to help you organize your Instance.
-        - Click **Advanced options** if you want to configure a [cloud-init configuration](/compute/instances/concepts/#cloud-init). Otherwise, leave these options at their default values.
-        - Verify the [SSH keys](/console/account/concepts/#ssh-key) that will give you access to your Instance.
-        - Verify the **Estimated cost** of your Instance, based on the specifications you chose.
+        - (Optional) Click **Advanced options** if you want to configure a [cloud-init configuration](/compute/instances/concepts/#cloud-init). Otherwise, leave these options at their default values.
```

Comment:
```suggestion
        - (Optional) Click **Advanced options** to configure a [cloud-init configuration](/compute/instances/concepts/#cloud-init). Otherwise, leave these options at their default values.
```